crazyclock.ino

```diff
@@ -80,13 +95,52 @@ void setup() {
   checkRTC(&lcd, &rtc);
 
   lcd.clear();
-  resetToRealTime();
-  myMillis = (millis() + tick);
+
+  currentSecond = rtc.getSecond();
+  Serial.print("Current second value: ");
+  Serial.println(currentSecond);
+  while (currentSecond == rtc.getSecond()) {
+    // waits until new second starts
+  }
+  newSecondStartedAtMillis = millis();
+  // increases value of current second
+  currentSecond++;
+  Serial.print("New second started at millis: ");
+  Serial.println(newSecondStartedAtMillis);
+
+  DateTime now = RTClib::now();
+  programStartedSeconds = now.unixtime();
```

Comment:
What this does do?

Reply:
`programStartedSeconds` is a variable that holds value of when the program started in "unix seconds" - seconds passed from midnight 1970-01-01. It is used later to calculate how long (in real time) the program is running. so that we can calculate new value for fake time:

> fakeTime = programStartedAt + (scalingFactor * timePassed)

Reply:
Should't we have 'programStartedSeconds' in better resolution? Msec?

Reply:
You are right! I have updated it to have both: [seconds and millis](https://github.com/The-Coobaz/crazyclock/pull/53/files#diff-ecf1bb4d938c070a787bf4a6b4a781d616f54a43102609cdd4fedf65c1daab5fR117):

```java
DateTime now = RTClib::now();
epochSecondsForStartPoint = currentSecond = now.unixtime();
millisForStartPoint = millis() - newSecondStartedAtMillis;
```